scripts/make/git.mk

```diff
@@ -15,6 +15,25 @@
 FULL_VERSION := $(shell git describe --long)
 TAG_VERSION  := $(firstword $(subst -, ,$(FULL_VERSION)))
 
+SUBMODULE_VERSION ?= latest
+
 ifeq (,$(FULL_VERSION))
 $(error "Version value could not be determined. Make sure you fetch the tags.")
 endif
+
+submodules: libraries/sky130_fd_sc_hd/$(SUBMODULE_VERSION)/.git libraries/sky130_fd_sc_hdll/$(SUBMODULE_VERSION)/.git libraries/sky130_fd_sc_hs/$(SUBMODULE_VERSION)/.git libraries/sky130_fd_sc_ms/$(SUBMODULE_VERSION)/.git libraries/sky130_fd_sc_ls/$(SUBMODULE_VERSION)/.git
+
+libraries/sky130_fd_sc_hd/%/.git: .gitmodules
```

Comment:
I depend on the `.gitmodules` file here so that they only get rechecked after the .gitmodules file is updated.